src/extension_utils/utils.ts

```diff
@@ -302,17 +301,30 @@ export const getTelemetryState = () => {
     .get("telemetry.enableTelemetry", true);
 };
 
-export const installPythonDependencies = async (context: vscode.ExtensionContext, pythonExecutable: string) => {
+export const installPythonDependencies = async (context: vscode.ExtensionContext, pythonExecutable: string, pathToLibs: string) => {
   let installed: boolean = false;
   try {
     vscode.window.showInformationMessage(CONSTANTS.INFO.INSTALLING_PYTHON_DEPENDENCIES);
-    const requirementsPath: string = getPathToScript(context, "out", "requirements.txt");
-    const pathToLibs: string = getPathToScript(context, "out", "python_libs");
+    
+    const requirementsPath: string = getPathToScript(context, CONSTANTS.FILESYSTEM.OUTPUT_DIRECTORY, "requirements.txt");
+
+    // run command to download dependencies to out/python_libs
     const { stdout } = await exec(`${pythonExecutable} -m pip install -r ${requirementsPath} -t ${pathToLibs}`);
-    console.info(stdout);
+    console.log(stdout);
```

Comment:
Just curious, why this was changed

Reply:
Oops, I'll change it back to console.info!